com.cubrid.common.ui/src/com/cubrid/common/ui/common/preference/JdbcManageComposite.java

```diff
@@ -139,7 +140,32 @@ public void save() {
 	private void createJdbcTableGroup(Composite composite) {
 		final String[] columnNameArr = new String[]{
 				Messages.tblColDriverVersion, Messages.tblColJarPath };
+		
 		TableViewerSorter sorter = new TableViewerSorter();
+		sorter.setColumnComparator(0, new Comparator<Object>(){
+			public int compare(Object o1, Object o2){
+				if(o1 instanceof String && o2 instanceof String){
+					String s1 = (String)o1;
+					String s2 = (String)o2;
+				
+					String[] version1Tokens = s1.substring(s1.lastIndexOf('-')+1).split("\\.");
+					String[] version2Tokens = s2.substring(s2.lastIndexOf('-')+1).split("\\.");
+					
+					int size = Math.min(version1Tokens.length, version2Tokens.length);
+					
+					for(int i = 0; i < size; i++){
+						Integer first = Integer.parseInt(version1Tokens[i]);
+						Integer second = Integer.parseInt(version2Tokens[i]);
+						if(first != second){
+							return second-first;
```

Comment:
@paulgheorghecristian 
Thank you, it's good for me. By the way, I'd like to improve more. Most people may want to see descending order by the jdbc version because latest version would be selected more frequently than old version. So, `first-second` and `version1Tokens.length - version2Tokens.length` may be better.

![ss](https://cloud.githubusercontent.com/assets/1791415/20931671/2a774d3e-bc15-11e6-8edc-073a59c87080.png)